test/framework/kube-events-exporter.go

```diff
@@ -69,8 +70,26 @@ type KubeEventsExporter struct {
 
 // CreateKubeEventsExporter creates kube-events-exporter deployment inside
 // of the specified namespace.
-func (f *Framework) CreateKubeEventsExporter(ns, exporterImage string) (*KubeEventsExporter, error) {
-	var finalizers []finalizerFn
+func (f *Framework) CreateKubeEventsExporter(t *testing.T) *KubeEventsExporter {
+	sa, err := MakeServiceAccount("../../manifests/kube-events-exporter-service-account.yaml")
```

Comment:
nit: could you define those paths as `const` and move to the top of the file? It may help later when paths or filenames change.